site/_config.yml

```diff
@@ -126,9 +126,9 @@ host:    127.0.0.1
 sourcerepourl: https://github.com/apache/incubator-edgent
 downloadsurl: /docs/downloads
 projurl: /
-#docsurl: /javadoc/latest
+docsurl: /javadoc/latest
 # above yields things like the following when running on a "jekyll serve" test server
```

Comment:
woops, sorry, just noticed your jira comment that the script is run after the site is built hence the msgs will still show up when running "jekyll serve" https://issues.apache.org/jira/browse/EDGENT-327?focusedCommentId=15812317&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15812317
If that's the case maybe add some commentary to README.md (where it describes running "jekyll serve") so folks know its safe to ignore those particular error msgs?